docs/pages/faq.mdx

```diff
@@ -113,7 +113,8 @@ Please refer to our [Networking](./reference/networking.mdx) guide.
 
 ## Does Teleport support authentication via OAuth, SAML, or Active Directory?
 
-Teleport offers this feature for the [Enterprise versions of Teleport](choose-an-edition/teleport-enterprise/introduction.mdx).
+Teleport offers this feature for the Teleport Enterprise and Teleport Enterprise Cloud editions.
+See [comparing the editions](choose-an-edition/introduction.mdx#access-controls) for more details.
```

Comment:
Would it make sense to link to this section within the same page, rather than having the user visit another page to see the same partial?

Reply:
I'd keep it this way. From there they can explore more of the edition information.

Reply:
Also worth noting that the edition comparison table comes before this question, so a reader who is going through the FAQ sequentially will have already seen it. Linking to another page will be pretty jarring. Maybe we can link to the "Introduction" page of "Choose an Edition" for more information and make it clear that we're doing this?

Reply:
What do you think of the comment above, @stevenGravy ?